src/Playwright/Helpers/TaskHelper.cs

```diff
@@ -204,6 +204,6 @@ private static async Task<bool> TimeoutTask(Task task, TimeSpan timeout)
 
     public static void IgnoreException(this Task task)
     {
-        _ = task.ContinueWith(t => t.Exception.Handle(_ => true), CancellationToken.None, TaskContinuationOptions.OnlyOnFaulted, TaskScheduler.Default);
+        _ = task.ContinueWith(t => t.Exception!.Handle(_ => true), CancellationToken.None, TaskContinuationOptions.OnlyOnFaulted, TaskScheduler.Default);
```

Comment:
whats the background for this? Since it didn't threw a warning before. Which .NET version are you using?

Reply:
I'm using .NET 7.0, I will recheck it in a moment.

Reply:
Ok it is actually the Resharper that is complaining at this line.
The null-forgiving operator stops them.
<img width="1033" alt="image" src="https://user-images.githubusercontent.com/6883328/230133451-32daea50-1abb-45e5-a371-826d5b705efd.png">

Reply:
Should I remove the change?

Reply:
Rolled back